logical/src/main/java/org/apache/drill/common/expression/FieldReference.java

```diff
@@ -1,4 +1,4 @@
-/**
```

Comment:
I have seen you changed '/**' to '/*' in several pull requests. What is the reason for that? I saw other Apache projects, e.g. Hive, use the previous style.